diff --git a/build/walletLink.js b/build/walletLink.js
index 6cdd44e38a9be056c16b7c2fa11f89853bff0bcc..2f18781bf97c7a88d7b55bdb7379509f084e67fe 100644
--- a/build/walletLink.js
+++ b/build/walletLink.js
@@ -17,7 +17,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.createUpdateHotspotUrl = exports.createSignHotspotCallbackUrl = exports.createLinkWalletCallbackUrl = exports.createWalletLinkUrl = exports.parseWalletLinkToken = exports.verifyWalletLinkToken = exports.makeAppLinkAuthToken = void 0;
 /* eslint-disable object-curly-newline */
 const address_1 = __importDefault(require("@helium/address"));
-const crypto_1 = require("@helium/crypto");
+const crypto_1 = require("@helium/crypto-react-native");
 const query_string_1 = __importDefault(require("query-string"));
 const date_fns_1 = require("date-fns");
 const types_1 = require("./types");
